code/modules/modular_armor/attachables.dm

```diff
@@ -51,7 +51,8 @@
 /obj/item/armor_module/attachable/better_shoulder_lamp/mark1
 	name = "\improper Mark 1 Baldur Light Amplification System"
 	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Substantially increases the power output of the Jaeger Combat Exoskeleton's mounted flashlight. Slows you down minorly."
-	power_boost = 3 /// The boost to armor shoulder light
+	power_boost = 4 /// The boost to armor shoulder light
```

Comment:
IIRC if you go any higher than 3 it starts causing popin so revert this please

Reply:
i mean the MK2 has 5

Reply:
ok that sfine then